modules/uhttp/filterchain_builder.go

```diff
@@ -52,7 +53,15 @@ type filterChainBuilder struct {
 
 func defaultFilterChainBuilder(host service.Host) filterChainBuilder {
 	fcb := newFilterChainBuilder(host)
-	return fcb.AddFilters(contextFilter(host), panicFilter(host), tracingServerFilter(host), authorizationFilter(host))
+	scope := host.Metrics().Tagged(modules.HTTPTags)
+	return fcb.AddFilters(
+		panicFilter{scope.Counter("panic")},
+		metricsFilter{scope},
+		tracingServerFilter{scope},
+		authorizationFilter{
+			authClient:  host.AuthClient(),
+			authCounter: scope.Counter("auth.fail"),
```

Comment:
One can use the prometheus as backend and the use '_' for qualified names: https://godoc.org/github.com/prometheus/client_golang/prometheus#Opts
Up to you anyway.